client/packages/executor/package.json

```diff
@@ -5,16 +5,17 @@
   "license": "MIT",
   "scripts": {
     "preinstall": "npx only-allow yarn",
-    "start": "EXECUTOR=example ts-node src/main.ts",
-    "dev": "LOG_LEVEL=debug EXECUTOR=example ts-node src/main.ts | npx pino-pretty -c -S",
+    "build": "tsup",
+    "start": "yarn build && EXECUTOR=example node dist/src/main.js",
+    "dev": "yarn build && LOG_LEVEL=debug EXECUTOR=example node dist/src/main.js | npx pino-pretty -c -S",
     "lint": "eslint --ext .ts src",
     "lint:report": "eslint src --ext .ts --output-file eslint_report.json --format json",
     "export": "LOG_LEVEL=debug yarn start >> logger.log",
     "fmt": "prettier --write src config tests",
     "build-docs": "npx typedoc && cp -R docs ../../../docs/executor",
-    "test": "mocha -r ts-node/register 'tests/**/*.ts'",
-    "test:attestion": "mocha -r ts-node/register 'tests/**/attest*.ts'",
-    "test-single": "mocha -r ts-node/register"
+    "test": "yarn build && mocha 'dist/tests/**/*.js'",
```

Comment:
test, start, etc should run yarn build
build should build

Reply:
Okay, see the commit [fc57e2f](https://github.com/t3rn/t3rn/pull/1091/commits/fc57e2f4c75aed2b08eb3889f0159fc368b0bbac).